SortDisplay.py

```diff
@@ -20,8 +21,7 @@ def __init__(self, algorithm, stop_event, options, compare=None):
         self.compare = compare
 
         self.stop = False
-        self.cstop = compare == None
-
+        self.cstop = True if compare is None else False
```

Comment:
Shall I revert it back to it's old version? I can't find another way to accomplish this in one line.